group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -5252,7 +5066,7 @@ private CoordinatorResult<Void, CoordinatorRecord> tryCompleteInitialRebalanceEl
         if (group.newMemberAdded() && remainingMs != 0) {
             // A new member was added. Extend the delay.
             group.setNewMemberAdded(false);
-            int newDelayMs = Math.min(classicGroupInitialRebalanceDelayMs, remainingMs);
+            int newDelayMs = Math.min(config.classicGroupInitialRebalanceDelayMs(), remainingMs);
             int newRemainingMs = Math.max(remainingMs - delayMs, 0);
```

Comment:
@dajac Given that `newDelayMs` is the value utilized by the scheduler, should we modify the expression to `Math.max(remainingMs - newDelayMs, 0)`?